wolftpm/tpm2_types.h

```diff
@@ -92,6 +93,13 @@ typedef int64_t  INT64;
 	#ifdef DEBUG_WOLFTPM
 		#include <stdio.h>
 	#endif
+
+    #include <wolfssl/version.h>
+    #if defined(LIBWOLFSSL_VERSION_HEX) && LIBWOLFSSL_VERSION_HEX < 0x03015004
+        /* The wc_HashFree was added in v3.15.4, so use stub to allow building */
+        #define wc_HashFree(h, t)
```

Comment:
in recent versions, the prototype for this function is
`int wc_HashFree(wc_HashAlg* hash, enum wc_HashType type)`

I'd suggest to define this as its hardcoded return value, because it should be legal to call it as in:
`ret = wc_HashFree(x,y);` when it not compiled in.

We should do:
`#define wc_HashFree(h, t)  (0)` to ignore calls when the symbol is not there